src/views/SystemIntake/Page1/index.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useState } from 'react';
+import { Field, FieldArray, FormikProps } from 'formik';
+import TextField from 'components/shared/TextField';
+import CheckboxField from 'components/shared/CheckboxField';
+import { DropdownField, DropdownItem } from 'components/shared/DropdownField';
+import cmsDivisionsAndOffices from 'constants/enums/cmsDivisionsAndOffices';
+import cmsGovernanceTeams from 'constants/enums/cmsGovernanceTeams';
+import { SystemIntakeForm } from 'types/systemIntake';
+
+type Page1Props = {
+  formikProps: FormikProps<SystemIntakeForm>;
+};
+
+const Page1 = ({ formikProps }: Page1Props) => {
+  const [isReqAndBusOwnerSame, setSameness] = useState(false);
```

Comment:
This piece of state is used for the `Same as requestor` checkbox. It indicates whether a business owner is the same person as the requestor. This field is being managed with component state rather than Formik because we are not actually sending this data. The checkbox acts as a simplified experience for the user so they don't need to fill out the same data multiple times.

Reply:
Nice! Can we rename "setSameness" to be more descriptive? maybe setReqAndBusOwnerSame